### PR TITLE
new tests to improve test coverage

### PR DESCRIPTION
**Pull request by halucka**

### CODE REVIEW

File: tests/test_extension.py [hunk 1]
```diff
@@ -14,7 +15,34 @@ def app():
     obs.init_app(app)
 
     @app.route("/login", methods=["GET"])
-    def login():
+    def loginHandler():
```
Comment (ericgazoni) [inline tests/test_extension.py]: I would tend to follow PEP8 here, and use `login_handler` instead

File: tests/test_extension.py [hunk 2]
```diff
@@ -14,7 +15,34 @@ def app():
...
+    app = Flask("demo")
+    app.config["TESTING"] = True
+    app.config["OBSERVE_AUTO_BIND_VIEWS"] = True
+    obs = Observability(hostname="somehost")
+    obs.init_app(app)
+    login_manager = LoginManager(app)
+    login_manager.init_app(app)
+
+    @app.route("/login", methods=["GET"])
+    def loginHandler():
```
Comment (ericgazoni) [inline tests/test_extension.py]: same comment as above about pep8

File: tests/test_extension.py [hunk 3]
```diff
@@ -14,7 +15,34 @@ def app():
     obs.init_app(app)
 
     @app.route("/login", methods=["GET"])
-    def login():
+    def loginHandler():
+        if request.form.get("username") == "bad":
+            abort(403)
+        return make_response("", 200)
+
+    @app.route("/error", methods=["GET"])
```
Comment (ericgazoni) [inline tests/test_extension.py]: I appreciate the need to generate and endpoint that triggers errors, but I don't see where you're calling it in your tests. Am I missing something?

Comment (halucka) [reply tests/test_extension.py]: I'm calling it in test_metrics_errors, but only for one error code 501. I originally thought to use it also for something from 1xx range and 3xx range, but those are not really errors and cannot be used with abort.

Comment (ericgazoni) [reply tests/test_extension.py]: sorry, my bad, I didn't read the test properly 😄

**Pull request merged**